main/zoe/guide/contracts/constantProductAMM.md

```diff
@@ -110,38 +110,17 @@ E(saraAtmPurse).deposit(atmRefund);
 ###  Creating a new Pool
 
 When the contract is first instantiated, there won't be any pools ready for
-trading. `addPool()` adds a new currency, which can then be funded.  (All
+trading. `addPoolInvitation()` adds a new currency, which can then be funded.  (All
```

Comment:
I don't think `addPoolInvitation()` adds a new currency. It just creates an invitation, right?